docs/user/contacts.md

```diff
@@ -0,0 +1,53 @@
+# Contacts
+
```

Comment:
Could you please write this document's description in one or two sentences, which will help users what contents this document describes?